src/Renderer/PointsMaterial.js

```diff
@@ -26,12 +26,19 @@ class PointsMaterial extends RawShaderMaterial {
         if (__DEBUG__) {
             this.defines.DEBUG = 1;
         }
+
+        this.updateUniforms();
     }
 
-    enablePicking(v) {
+    enablePicking(pickingMode) {
         // we don't want pixels to blend over already drawn pixels
-        this.blending = v ? NoBlending : NormalBlending;
-        this.uniforms.pickingMode.value = v;
+        this.blending = pickingMode ? NoBlending : NormalBlending;
+        this.uniforms.pickingMode.value = pickingMode;
+    }
+
+    updateUniforms() {
+        // if size is null, switch to autosizing using the canvas height
+        this.uniforms.size.value = (this.size > 0) ? this.size : -this.scale * window.innerHeight;
```

Comment:
@mbredif this isn't the canvas size. Are you sure it displays correctly when the canvas is not "fullscreen"?

Reply:
I am sure it does not, but that is how it was coded (I just made it slightly better by taking the current window.innerHeight rather than the one at material creation time). 

If we trust these pieces of code:
https://github.com/iTowns/itowns/blob/f18548f4bb907dea96efdc620acb0863da5fca88/src/Process/PointCloudProcessing.js#L96-L100 
https://github.com/iTowns/itowns/blob/f18548f4bb907dea96efdc620acb0863da5fca88/src/Process/PointCloudProcessing.js#L59-L66

then something like this might be better :
```
 this.uniforms.size.value = (this.size > 0) ? this.size : -camera.preSSE * layer.metadata.spacing / Math.pow(2, elt.name.length);
```
but that would require passing the necessary parameters and necessitate some testing :). maybe in another PR ?

Reply:
>  that is how it was coded  

oh right, it was there before this PR, my bad... I was thinking of using at least the canvas height instead of the windows height at least. But yeah, another PR another day :-) Thanks for pointing this to me!